src/main.rs

```diff
@@ -265,67 +280,76 @@ async fn main() -> anyhow::Result<()> {
                 }
             }
         }; // drop lock
-        log::debug!("Computed median: {:#?}", rate);
+        log::debug!("Computed median: {}", rate);
 
+        // TODO: SHouldn't this be comparing actual rates, not ones off by a million?
```

Comment:
The relative difference should be same, if the values are off by the same factor?
(Though this did make me realize that the initial `prev_rate` was microCCD/euro, while `rate` is CCD/Euro)

Reply:
Yes, but it is still a bit dirty. At least comment to be explicit that this is happening.

Reply:
Okay, I realize now, I could just change the unit earlier, so that it uses the microCCD/euro rates instead.